src/main/kotlin/no/nav/syfo/personstatus/infrastructure/clients/veileder/VeilederDTO.kt

```diff
@@ -1,6 +1,6 @@
 package no.nav.syfo.personstatus.infrastructure.clients.veileder
 
 data class VeilederDTO(
-    val enabled: Boolean? = null,
+    val enabled: Boolean,
```

Comment:
Kunne ønsket meg en `isEnabled` for å tydeliggjøre at det er en `Boolean`, men nå er vel dette avhengig av apiet i `syfoveileder` så bare la det være 😊